VortexEngine/src/VortexConfig.h

```diff
@@ -174,6 +174,13 @@
 // produced by fastled's hsv to rgb 'rainbow'
 #define RGB_TO_HSV_ALGORITHM   1
 
+// IR Receiver Time-out Duration
```

Comment:
I like to add (ms) at the end of the title for timing configs so people quickly know what unit the timing is